packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -766,7 +782,10 @@ class _DropdownMenuState<T> extends State<DropdownMenu<T>> {
       if (widget.searchCallback != null) {
         currentHighlight = widget.searchCallback!(filteredEntries, _localTextEditingController!.text);
       } else {
-        currentHighlight = search(filteredEntries, _localTextEditingController!);
+        final bool shouldUpdateCurrentHighlight = _shouldUpdateCurrentHighlight(filteredEntries, _localTextEditingController!);
```

Comment:
Any reason for passing _localTextEditingController as a parameter? It is an instance variable so it is visible.

Reply:
Sure, let me do the change